src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/LightCompiler.cs

```diff
@@ -1509,14 +1504,14 @@ private void CompileSwitchExpression(Expression expr)
                     return;
                 }
 
-                TypeCode switchType = System.Dynamic.Utils.TypeExtensions.GetTypeCode(node.SwitchValue.Type);
+                TypeCode switchType = node.SwitchValue.Type.GetTypeCode();
```

Comment:
Tracking netstandard1.7 build for this project in https://github.com/dotnet/corefx/issues/13052.